index.html

```diff
@@ -8,17 +8,52 @@
     <title>Book Tracker</title>
     <link rel="stylesheet" href="./css/style.css">
 </head>
+
 <body>
-    <div class="heading-container">
-        <h2>Your Reading List:</h2>
-    </div>
-    <div id="collection" class="reading-collection"></div>
-    <div class="input-section"> 
-        <input id="title" class="input-title" placeholder="Book Title">
-        <input id="author" class="input-author" placeholder="Book Author">
-        <input type="submit" id="btnAdd" class="button">
-        <p class="error-message" id="error"></p>
-    </div>
+    <header class="nav-section">
+        <!-- <div></div> -->
```

Comment:
- [ ] OPTIONAL - Kindly remove this commented code line, as commented code makes the project untidy, you can use GitHistory to refer to previous code versions.